### PR TITLE
Added new project upgrade flow to the docs

### PR DESCRIPTION
**Pull request by jonat123**

### CODE REVIEW

File: umbraco-cloud/set-up/project-settings/upgrade-umbraco-cloud-plan.md [hunk 1]
```diff
@@ -27,17 +27,26 @@ In the menu, you can find a tab called _"Upgrade plan"_.
...
 
-![Upgrade plane step2](../images/upgrade\_plan.png)
+![Upgrade plane step2](../images/umbraco-cloud-plans-new.png)
 
 * If you are on a Starter plan you can upgrade to the Standard and the Professional plan.
 * If you are on the Standard plan you can upgrade to the Professional plan.
 
-To upgrade to the plan of your choosing, click on the green button for the plan that you want to upgrade to and then your project will be upgraded.
+Follow the below steps to upgrade your plan:
+1. Click on the button saying **"Select plan"** to choose the plan you want to upgrade to.
```
Comment (eshanrnh) [inline umbraco-cloud/set-up/project-settings/upgrade-umbraco-cloud-plan.md:36]:
```suggestion
1. Click on the **Select plan** button to choose the plan you want to upgrade to.
```

File: umbraco-cloud/set-up/project-settings/upgrade-umbraco-cloud-plan.md [hunk 2]
```diff
@@ -27,17 +27,26 @@ In the menu, you can find a tab called _"Upgrade plan"_.
...
+2. In the next step, you can choose to upragde with to a **dedicated option**.
```
Comment (eshanrnh) [inline umbraco-cloud/set-up/project-settings/upgrade-umbraco-cloud-plan.md:37]:
```suggestion
2. Choose to upgrade to a **dedicated option** in the next step.
```

File: umbraco-cloud/set-up/project-settings/upgrade-umbraco-cloud-plan.md [hunk 3]
```diff
@@ -27,17 +27,26 @@ In the menu, you can find a tab called _"Upgrade plan"_.
...
+   ![Dedicated option when upgrading](../images/dedicated-option.png)
+3. In the last step, review the **summary** to make sure that everything selected is correct.
```
Comment (eshanrnh) [inline umbraco-cloud/set-up/project-settings/upgrade-umbraco-cloud-plan.md:39]:
```suggestion
3. Review the **Summary** to make sure that everything selected is correct in the last step.
```

File: umbraco-cloud/set-up/project-settings/upgrade-umbraco-cloud-plan.md [hunk 4]
```diff
@@ -27,17 +27,26 @@ In the menu, you can find a tab called _"Upgrade plan"_.
...
+![Upgrade summary](../images/upgrade-summary.png)
+
+Once you click the button **"Upgrade Project"** the project will be upgrade to the new plan and if selected to a dedicated server.
```
Comment (eshanrnh) [inline umbraco-cloud/set-up/project-settings/upgrade-umbraco-cloud-plan.md:42]:
```suggestion
Once you click the **Upgrade Project** button, the project will be upgraded to the new plan and if selected to a dedicated server.
```

**Review by eshanrnh — COMMENTED** [review]
Looks good to me 🙌 I've made some suggestions based on the Action first concept in the list 🤞

File: umbraco-cloud/set-up/project-settings/upgrade-umbraco-cloud-plan.md (hunk 2 above)
Comment (eshanrnh) [reply umbraco-cloud/set-up/project-settings/upgrade-umbraco-cloud-plan.md:37]: Ah ok.. So maybe you can rephrase it to [Optional] Choose to upgrade to a **dedicated option** in the next step.
```suggestion
2. _[Optional]_ Choose to upgrade to a **dedicated option** in the next step.
```

Comment (jonat123) [reply umbraco-cloud/set-up/project-settings/upgrade-umbraco-cloud-plan.md:37]: Yeah, I like that better 😊
